src/CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ if(LMMS_HAVE_MP3LAME)
 	list(APPEND EXTRA_LIBRARIES mp3lame::mp3lame)
 endif()
 
+IF(LMMS_USE_MINGW_STD_THREADS)
+	list(APPEND EXTRA_LIBRARIES mingw_stdthreads)
+ENDIF()
```

Comment:
The command casing here is inconsistent. I would prefer lowercase, as that's now the modern preferred CMake style, but uppercase is also acceptable if you want to maintain consistency with more of the code.

Reply:
> The command casing here is inconsistent. I would prefer lowercase, as that's now the modern preferred CMake style, but uppercase is also acceptable if you want to maintain consistency with more of the code.

Not to mention that in certain editors, you don't get extra help on the various CMake commands if its all in uppercase, but in lowercase. I was using mostly using uppercase rather than lowercase for consistency, but I'll lean towards the latter for this one.